assets/scss/_navbar_project.scss

```
@@ -260,4 +260,248 @@ $ecosystem-box-glow: inset 0 0em 4em #ebc01766, 0 0 0 2px #ebc01766, 0.3em 0.3em
+      &.active {
+        color: $primary !important;
+        text-shadow: none !important;
+      }
+    }
+
+    #resourcesDropdown {
+      color: #3c494f !important;
+      text-shadow: none !important;
+
+      .down-icon svg {
+        fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)